tools/walletextension/frontend/src/components/health-indicator.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from "react";
+import { Badge, badgeVariants } from "./ui/badge";
+import useGatewayService from "@/services/useGatewayService";
+import { Skeleton } from "./ui/skeleton";
+
+const HealthIndicator = () => {
+  const { getTestnetStatus } = useGatewayService();
+  const [loading, setLoading] = React.useState(false);
+  const [status, setStatus] = React.useState<boolean>();
+
+  const testnetStatus = async () => {
+    setLoading(true);
+    try {
+      const status = await getTestnetStatus();
+      return status;
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    testnetStatus().then((res) => setStatus(res?.result));
+  }, []);
```

Comment:
./src/components/health-indicator.tsx
25:6  Warning: React Hook useEffect has a missing dependency: 'testnetStatus'. Either include it or remove the dependency array.  react-hooks/exhaustive-deps